lib/tomlrb/local_date_time.rb

```diff
@@ -0,0 +1,66 @@
+module Tomlrb
+  class LocalDateTime
+    def initialize(year, month, day, hour, min, sec)
+      @time = Time.new(year, month, day, hour, min, sec, '-00:00')
+      @sec = sec
+    end
+
+    def year
```

Comment:
I think we could do
```ruby
def_delegators :@time, :year, :month, :day, :hour, :min, :sec, :usec, :nsec
```

and avoid quite a lot of repetition

Reply:
Okay, I will rewrite.

Reply:
Done.